Constants.java

```diff
@@ -0,0 +1,22 @@
+package com.company;
+
+public class Constants {
+
+    public static final String FILE_TYPE_TXT = ".txt";
+    public static final String FILE_TYPE_JSON = ".json";
+    public static final String REMOVE_SPECIAL_CHAR = "[^a-zA-Z0-9 -]";
+    public static final String SINGLE_SPACE = "";
+    public static final String ERROR_MESSAGE_INCORRECT_NUMBER_OF_INPUTS = "The number of inputs must not exceed or subceed two";
+    public static final String ERROR_MESSAGE_FILE_PATH_EMPTY_OR_NULL = "File path cannot be empty";
+    public static final String ERROR_MESSAGE_SEARCH_WORD_EMPTY_OR_NULL = "Search word cannot be empty";
+    public static final String ERROR_MESSAGE_EMPTY_FILE = "File does not contain any data";
+    public static final String ERROR_MESSAGE_FILE_NOT_FOUND = "File does not exist";
+    public static final String ERROR_MESSAGE_FILE_FORMAT_NOT_SUPPORTED = "File format not supported";
+    public static final String ERROR_MESSAGE_WORD_NOT_FOUND = "Word not found";
+    public static final String SUCCESS = "SUCCESS";
+    public static final String FAILURE = "FAILURE";
+    public static final String insertSql = "INSERT INTO audit VALUES(?,?,now(),?,?,?)";
```

Comment:
```suggestion
    public static final String INSERT_SQL_STATEMENT = "INSERT INTO audit VALUES(?,?,now(),?,?,?)";
```
ALL CAPS for constants

Reply:
Made the change.